lib/perl5/site_perl/DataMerge.pm

```diff
@@ -125,12 +125,20 @@ sub compare_fastq_bam_reads {
 	}
 
 	exit();
-	#time /Volumes/cluster/DistMap_v1.2.1/executables/samtools-0.1.19/samtools index /Volumes/cluster/Ray/BGI_143b/DistMap_output_Paired_end_reads.bam
-	#time /Volumes/cluster/DistMap_v1.2.1/executables/samtools-0.1.19/samtools idxstats /Volumes/cluster/Ray/BGI_143b/DistMap_output_Paired_end_reads.bam | awk '{i+=$3+$4} END {print i}'
-
-	my $samtools = $path."executables/samtools-0.1.19/samtools";
-	my $command1 = "$samtools index $final_output_file";
-	my $command2 = "$samtools idxstats $final_output_file | awk '{i+=\$3+\$4} END {print i}' > $mapped_file_std";
+	
+	## TODO - this code is not really used anywhere: the exit before would end the subroutine and in addition compare_fastq_bam_reads
```

Comment:
@robmaz - this is quite interesting: maybe it is better to just remove the code/sub-routine